app/views/components/docs/translation-nav.yml

```diff
@@ -0,0 +1,21 @@
+name: Translation Nav
+description: A navigation item for available language translations.
+body: The active property should be used to indicate the currently selected language.
+accessibility_criteria: |
+  The translation nav must:
+  - accept focus
```

Comment:
For the markdown to render as a list there needs to be a line break before the first `- item`